pymc3/plots/energyplot.py

```diff
@@ -0,0 +1,61 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from .utils import fast_kde
+
+def energyplot(trace, kind='kde', figsize=None, ax=None, legend=True, lw=0, alpha=0.5, **kwargs):
```

Comment:
I think the KDE is easier to read because it smooths out the histogram. I can experiment with filling it with color, though.